github-api/src/main/java/org/eclipse/egit/github/core/service/PullRequestService.java

```diff
@@ -229,6 +243,21 @@ public PullRequest getPullRequest(IRepositoryIdProvider repository, int id)
 		return params;
 	}
 
+	/* ED ED */
+    private Map<String, String> createPrReviewMap(Review request) {
```

Comment:
This appears to have only one usage which is commented out. You should remove it if it's no longer needed or fix the usage.